sapi/fpm/config.m4

```diff
@@ -605,6 +605,12 @@ if test "$PHP_FPM" != "no"; then
     ])
   fi
 
+  AC_CHECK_HEADERS([selinux/selinux.h])
+  AC_CHECK_LIB(selinux, security_setenforce, [
+    PHP_ADD_LIBRARY(selinux)
+    AC_DEFINE(HAVE_SELINUX, 1, [ SElinux available ])
+  ],[])
```

Comment:
I think we should have this optional in the same way as we have it for apparmor otherwise it's kind of "automagic" library dependency as described in this PR: https://github.com/php/php-src/pull/2989